src/pages/photos/_id.vue

```diff
@@ -62,15 +62,45 @@ const PhotoDetailPage = {
       socialSharingEnabled: featureFlags.socialSharing,
     }
   },
-  computed: mapState({
-    relatedImages: 'relatedImages',
-    filter: 'query.filter',
-    images: 'images',
-    imagesCount: 'imagesCount',
-    query: 'query',
-    tags: 'image.tags',
-    image: 'image',
-  }),
+  computed: {
+    fullLicenseName() {
+      const license = this.image.license
+      const version = this.image.license_version
+
+      if (license) {
+        return license.toLowerCase() === 'cc0'
+          ? `${license} ${version}`
+          : `CC ${license} ${version}`
+      }
+      return ''
+    },
+    ...mapState({
+      relatedImages: 'relatedImages',
+      filter: 'query.filter',
+      images: 'images',
+      imagesCount: 'imagesCount',
+      query: 'query',
+      tags: 'image.tags',
+      image: 'image',
+    }),
+  },
+  head() {
+    return {
+      title: `${this.image.title} by ${this.image.creator} | Free Image on Openverse`,
```

Comment:
Free sounds like something to do with price or royalty ("as in beer") rather than something to do with the license ("as in speech").

Reply:
Agreed, and I'm torn if that's a god thing or a bad thing. At the end of the day, folks are looking for "free" as in beer content. It does feel a little gimmicky though.